src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -53,12 +76,43 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
           class="markdown ${classMap({
             "no-header": !this._config.title,
           })}"
-          .content="${this._config.content}"
+          .content="${this._content}"
         ></ha-markdown>
       </ha-card>
     `;
   }
 
+  private async _connect() {
+    if (!this._unsubRenderTemplate && this._hass && this._config) {
+      this._unsubRenderTemplate = subscribeRenderTemplate(
+        this._hass.connection,
+        (result) => {
+          this._content = result;
+        },
+        {
+          template: this._config.content,
+          entity_ids: this._config.entity_id,
+        }
+      );
+      this._unsubRenderTemplate.catch(() => {
+        this._content = this._config!.content;
```

Comment:
Also set `this._unsubRenderTemplate = undefined`

Reply:
Ah yes. Thank you.